lib/time.rb

```diff
@@ -0,0 +1,7 @@
+class Time
```

Comment:
I like how you have separated this out, it makes it more readable in the code where you are using it!